about/reports/2020_downloads_top_250_institutions.md

```diff
@@ -0,0 +1,264 @@
+| 214 | uni-siegen.de | 0.08% | 25474                                    |
+| 215 | hokudai.ac.jp | 0.08% | 25263                                    |
+| 216 | polimi.it | 0.08% | 25161                                        |
+| 217 | metu.edu.tr | 0.08% | 24846                                      |
+| 218 | jinr.ru | 0.08% | 24487                                          |
+| 219 | csic.es | 0.08% | 24373                                          |
+| 220 | nao.ac.jp | 0.08% | 24366                                        |
+| 221 | fu-berlin.de | 0.08% | 24155                                     |
+| 222 | uni-leipzig.de | 0.08% | 24118                                   |
+| 223 | ncku.edu.tw | 0.08% | 23941                                      |
+| 224 | utl.pt | 0.08% | 23938                                           |
+| 225 | upv.es | 0.08% | 23660                                           |
+| 226 | syracuse.edu | 0.07% | 23488                                     |
+| 227 | usst.edu.cn | 0.07% | 23352                                      |
+| 228 | iastate.edu | 0.07% | 23000                                      |
+| 229 | cnrs.fr | 0.07% | 22992                                          |
+| 230 | bath.ac.uk | 0.07% | 22963                                       |
+| 231 | ifpan.edu.pl | 0.07% | 22647                                     |
+| 232 | jyu.fi | 0.07% | 22283                                           |
+| 233 | ipm.ac.ir | 0.07% | 22117                                        |
+| 234 | keio.ac.jp | 0.07% | 22049                                       |
+| 235 | lut.ac.uk | 0.07% | 21859                                        |
+| 236 | ucm.es | 0.07% | 21841                                           |
+| 237 | adelaide.edu.au | 0.07% | 21605                                  |
+| 238 | umontreal.ca | 0.07% | 21523                                     |
+| 239 | tu-ilmenau.de | 0.07% | 21465                                    |
+| 240 | unipi.it | 0.07% | 21456                                         |
+| 241 | unifi.it | 0.07% | 21309                                         |
+| 242 | okstate.edu | 0.07% | 21139                                      |
+| 243 | uni-bremen.de | 0.07% | 20927                                    |
+| 244 | au.dk | 0.07% | 20906                                            |
+| 245 | oregonstate.edu | 0.07% | 20643                                  |
+| 246 | tu-chemnitz.de | 0.07% | 20641                                   |
+| 247 | leeds.ac.uk | 0.07% | 20588                                      |
+| 248 | pd.infn.it | 0.06% | 20417                                       |
+| 249 | roma1.infn.it | 0.06% | 20320                                    |
+| 250 | univ-lyon1.fr | 0.06% | 20101                                    |
+
+(\*) Some institutions' downloads come from multiple domains which have been combined to report an aggregate number:
```

Comment:
If there are no * data this line could be removed
```suggestion

```

Reply:
@afromme does this look OK?

Reply:
@bdc34 or @bmaltzan can you confirm which, if any, domains' downloads were combined and mark them with an asterisk?

Reply:
In recent years I would guess that this list has been just copied and pasted from the previous year. Here is the list generated in the way it would have been in 2016. It doesn't include Max Plank or IP range based institutions:

```cnrs.fr includes downloads from cnrs-orleans.fr
fbk.eu-it includes downloads from fbk.eu
h-brs.de includes downloads from fh-bonn-rhein-sieg.de
ictp.it includes downloads from ictp.trieste.it
inaf.it includes downloads from astro.it
ku.dk includes downloads from dark-cosmology.dk
pmf.hr includes downloads from math.hr, gfz.hr
pnnl.gov includes downloads from pnl.gov
snl.gov includes downloads from sandia.gov
tkk.fi includes downloads from hut.fi, aalto.fi
tudelft.nl includes downloads from tudelft.net
uac.bj.refer.org-be includes downloads from uac.bj.refer.org
umk.pl includes downloads from uni.torun.pl
uni-muenchen.de includes downloads from lmu.de
unifesp.br includes downloads from epm.br
desy.de includes downloads from ifh.de
illinois.edu includes downloads from uiuc.edu
inaf.it includes downloads from astro.it
manchester.ac.uk includes downloads from man.ac.uk
riken.jp includes downloads from riken.go.jp
stonybrook.edu includes downloads from sunysb.edu
sydney.edu.au includes downloads from usyd.edu.au
tu-dortmund.de includes downloads from uni-dortmund.de
tu-graz.ac.at includes downloads from tugraz.at
tudelft.nl includes downloads from tudelft.net
tum.de includes downloads from tu-muenchen.de
ucsd.edu includes downloads from sdsc.edu
uni-saarland.de includes downloads from uni-sb.de
upmc.fr includes downloads from jussieu.fr
jhu.edu includes downloads from jhmi.edu
bristol.edu includes downloads from bris.ac.uk
toronto.edu includes downloads from utoronto.ca
tkk.fi includes downloads from aalto.fi, hut.fi
uni-karlsruhe.de includes downloads from fzk.de, kit.edu
```